packages/block-editor/src/components/link-control/link-overview.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, ExternalLink } from '@wordpress/components';
+import { filterURLForDisplay, safeDecodeURI } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { ViewerSlot } from './viewer-slot';
+
+export default function LinkOverview( { value, onEditClick } ) {
+	const displayURL =
+		( value && filterURLForDisplay( safeDecodeURI( value.url ) ) ) || '';
+
+	return (
+		<div
+			aria-label={ __( 'Currently selected' ) }
+			aria-selected="true"
+			className={ classnames( 'block-editor-link-control__search-item', {
+				'is-current': true,
+			} ) }
+		>
+			<span className="block-editor-link-control__search-item-header">
+				<ExternalLink
+					className="block-editor-link-control__search-item-title"
+					href={ value.url }
+				>
+					{ ( value && value.title ) || displayURL }
+				</ExternalLink>
+				{ value && value.title && (
+					<span className="block-editor-link-control__search-item-info">
+						{ displayURL }
+					</span>
+				) }
+			</span>
+
+			<Button
+				isSecondary
+				onClick={ () => onEditClick() }
+				className="block-editor-link-control__search-item-action"
+			>
+				{ __( 'Edit' ) }
+			</Button>
+			<ViewerSlot fillProps={ value } />
```

Comment:
What is this `ViewerSlot`? It appears it did not exist prior to this refactor and the PR description does not mention it.

Reply:
It did exist, I just moved it over from another file. If you're using cmd+f on the files tab of this PR, the largest file is folded by default which is why you can't find it.